Source/RealmResultsController.swift

```diff
@@ -23,9 +23,40 @@ public enum RealmResultsChangeType: String {
 }
 
 public protocol RealmResultsControllerDelegate: class {
+    
+    /**
+    Notifies the receiver that the fetched results controller is about to start processing of one or more changes due to an add, remove, move, or update.
+    
+    :param: controller The fetched results controller that sent the message.
```

Comment:
change `fetched results controller` to `realm results controller` everywhere please :dancer: